### PR TITLE
feat(vrl stdlib): function to parse aws classic load balancer logs

### PR DESCRIPTION
**Pull request by carnei-ro**

### CODE REVIEW

File: lib/vrl/stdlib/src/parse_aws_clb_log.rs [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+    macro_rules! field {
+        ($name:expr, $($pattern:pat_param)|+) => {
+            field_raw!($name, preceded(char(' '), take_while1(|c| matches!(c, $($pattern)|+))))
+        };
+    }
+    macro_rules! field_parse {
+        ($name:expr, $($pattern:pat_param)|+, $type:ty) => {
+            field_raw!($name, map_res(preceded(char(' '), take_while1(|c| matches!(c, $($pattern)|+))), |s: &str| s.parse::<$type>()))
+        };
+    }
```
Comment (StephenWakely) [inline lib/vrl/stdlib/src/parse_aws_clb_log.rs:146]: These macro rules appear to be the same as those in `parse_aws_alb_log`, would it be possible to move those into a shared file and reuse them here?

**Review by StephenWakely — CHANGES_REQUESTED** [review]
Hi, many thanks for this. Sorry for the delay in getting it reviewed.

I have one comment around reusing some code, and it needs a conflict with master resolved.

**Comment by StephenWakely** [comment]
@carnei-ro Could you hold for a bit on making these changes. We are currently discussing amongst ourselves if there is a better way we can manage all these specific parsing functions, so not sure at present if we will move forward with this..

**Comment by fuchsnj** [comment]
Hey @carnei-ro, we discussed this with the team and decided we can move forward with the parsing function as-is. Are you still available to finish this up?

**Comment by fuchsnj** [comment]
I apologize for the inconvenience, but VRL has been moved to a [separate repo](https://github.com/vectordotdev/vrl), and this PR will need to be moved over there.

**Pull request closed without merging**